src/index.html

```diff
@@ -186,89 +190,95 @@ <h1 class="text-xl/7 font-heading font-[375]" data-i18n="site_title">
         </div>
       </section>
     </main>
-    <footer class="bg-gray-100">
+    <footer
+      class="bg-gray-100"
+      aria-label="Go to GC Notify"
+      data-i18n="back_button"
```

Comment:
Same issue here.  Remove these 2 `data-i18n` attributes and it should be good to go.